### PR TITLE
update ux ,create score and progress bar

### PR DESCRIPTION
**Pull request by RubaMansour**

### CODE REVIEW

File: src/views/answerView.js [hunk 1]
```diff
@@ -2,22 +2,55 @@
...
+  element.type = 'button'; // Prevents default form submission behavior if inside a form
 
   const inputType = isMultiple ? 'checkbox' : 'radio';
 
+  // Add hidden input element for tracking selected answers
   element.innerHTML = String.raw`
-    <label>
-      <input type="${inputType}" name="answer" value="${key}" />
+    <label class="answer-label">
+      <input type="${inputType}" name="answer" value="${key}" style="display: none;" />
```
Comment (durw4rd) [inline src/views/answerView.js:19]: Hi Ruba, hiding the <input> element means it's not possible to tell whether the question is a single- or multiple choice. It's not clear to me why would you want to do this - does it cause any issues with the progress bar when the input element is visible?

Comment (RubaMansour) [reply src/views/answerView.js:19]: I will restore it to its previous state.

**Comment by op117** [comment]
see how the choice of answers works in the main for questions with multiple choice answers.
image
and how it works in your implementation 
image

**Review by op117 — APPROVED** [review]
did it

**Pull request merged**